iotdb-client/jdbc/pom.xml

```diff
@@ -172,9 +172,13 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
+                <version>3.4.2</version>
```

Comment:
You can upgrade this version in the parent pom.
https://github.com/apache/iotdb/blob/5f8b2c08ac91ec6f0d7042f39192c90cd173198f/pom.xml#L888